StripePaymentSheet/StripePaymentSheet/Source/Internal/Link/Utils/LinkPopupURLParser.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  LinkPopupURLParser.swift
+//  StripePaymentSheet
+//
+
+import Foundation
+import StripePayments
+
+struct LinkResult {
+    enum LinkStatus: String {
+        case complete
```

Comment:
Feels awkward to have a single case for an enum -- feels like something is missing here?

Reply:
Yeah, we haven't added the other cases yet.